2. Services/2.1 Foundations/2.1 Foundations.md

```diff
@@ -690,11 +698,11 @@ private async ValueTask<Student> TryCatch(ReturningStudentFunction returningStud
 	}
 	catch (InvalidStudentException invalidStudentInputException)
 	{
-		throw CreateAndLogValidationException(invalidStudentInputException);
+		throw await CreateAndLogValidationExceptionAsync(invalidStudentInputException);
 	}
 }
 
-private StudentValidationException CreateAndLogValidationException(Xeption exception)
+private async ValueTask<StudentValidationException> CreateAndLogValidationExceptionAsync(Xeption exception)
```

Comment:
missing await line 711:
`    await this.loggingBroker.LogErrorAsync(studentValidationException);`

Reply:
Fixed